docs/source/getting-started/installation.rst

```diff
@@ -17,6 +17,9 @@ Install SkyPilot using pip:
   $ # pip install "skypilot[lambda]"
   $ # pip install "skypilot[all]"
 
+.. note::
+
+    There is no native build for python version <3.8 on Apple Silicon. Apple Silicon-based devices should create the conda environment with python version >=3.8.
```

Comment:
Nice catch! How about folding these lines into L8-10:
```
  $ # SkyPilot requires python >= 3.6. For Apple Silicon, use >= 3.8.
  $ # Recommended: use a new conda env to avoid package conflicts.
  $ conda create -y -n sky python=3.8
```

Reply:
Just made the changes and made a new commit!